client/src/game/rules/clues.ts

```diff
@@ -1,19 +1,9 @@
+/* eslint-disable import/prefer-default-export */
```

Comment:
is this because there is only 1 exported function now?

Reply:
Yes, for now. It's a bit annoying.